Form-Controls/index.html

```diff
@@ -14,14 +14,67 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+    
+
+  <section>
+        
+<label for="name">enter your name</label>
```

Comment:
Have you installed Prettier? The style guide can help you do this: https://syllabus.codeyourfuture.io/guides/code-style-guide#using-prettier-to-format-code-automatically

Reply:
Thank you sally, i will work on it.